apps/ideaspace/src/components/common/IdeaCard/IdeaCard.js

```diff
@@ -119,13 +120,26 @@ function IdeaCard({ cards, cardType }) {
         <IdeaCardTag status={tagContent} />
       </atoms.Box> */}
         <div>
-          <IdeaCardImg cardId={cards.id} cardImg={cards.imgSrc} />
+          {cards.ideaImage && cards.ideaImage?.data?.attributes?.medium_url ? (
```

Comment:
We are already using `cards.ideaImage?`, do we still need the `cards.ideaImage` check?